### PR TITLE
[FIX] Solve weave/rest-smtp-sink: Docker image manifest v2 schema 1 deprecation issue

### PR DESCRIPTION
**Pull request by quantranhong1999**
cf: https://docs.docker.com/engine/deprecated/#pushing-and-pulling-with-image-manifest-v2-schema-1

Solution: https://hub.docker.com/r/quanth99/rest-smtp-sink

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]
Not optimal I think but ok. Good catch and thanks for this fix :)

File: server/testing/src/main/java/org/apache/james/util/docker/Images.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 package org.apache.james.util.docker;
 
 public interface Images {
-    String FAKE_SMTP = "weave/rest-smtp-sink:latest";
+    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0";
```
Comment (chibenwa) [inline server/testing/src/main/java/org/apache/james/util/docker/Images.java:23]: If you can comment the URL to the Dockerfile project on github then that's perfect!

Comment (quantranhong1999) [reply server/testing/src/main/java/org/apache/james/util/docker/Images.java]: done

File: server/testing/src/main/java/org/apache/james/util/docker/Images.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 package org.apache.james.util.docker;
 
 public interface Images {
-    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0";
+    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0"; // Original Dockerfile: https://github.com/ambled/rest-smtp-sink/blob/master/Dockerfile
```
Comment (chibenwa) [inline server/testing/src/main/java/org/apache/james/util/docker/Images.java:23]: No I mean the docker file of `quanth99/rest-smtp-sink:1.0`

Comment (quantranhong1999) [reply server/testing/src/main/java/org/apache/james/util/docker/Images.java:23]: That one I documented in https://hub.docker.com/r/quanth99/rest-smtp-sink docs

**Review by chibenwa — APPROVED** [review]

**Pull request merged**